api.wordpress.org/public_html/patterns/1.0/tests/test-index.php

```diff
@@ -81,19 +83,12 @@ public function get_term_slugs( $patterns ) {
 	 * @group e2e
 	 */
 	public function test_browse_all_patterns() : void {
-		$response = send_request( '/patterns/1.0/?per_page=100' );
+		$response = send_request( '/patterns/1.0/?per_page=20' );
```

Comment:
Decreased this to 20 to prevent the API response from timing out. The number doesn't matter as much as getting a set number. 20 is also arbitrary.

Reply:
in the wpcut tests that were added some known patterns were used with structure that did break requests - I wonder if we should either do similar here, or have a mix of known and random patterns?

Reply:
> some known patterns were used with structure that did break requests

Which known patterns are breaking requests? do you mean the category_slugs thing? This particular test would not catch that, but the other tests (that run `assertResponseHasPattern`) will check individual pattern return types (like `test_results_limited_to_requested_locale` &  `test_browse_patterns_by_category`).

Reply:
it was pattern id `199` which caused the errors - unsure if we want to do it statically or not like: https://api.wordpress.org/patterns/1.0/?include=199,229080,446939,482686,278112